source/base/mpi.cc

```diff
@@ -187,6 +187,7 @@ namespace Utilities
     {
       const unsigned int myid    = Utilities::MPI::this_mpi_process(mpi_comm);
       const unsigned int n_procs = Utilities::MPI::n_mpi_processes(mpi_comm);
+      MPI_Status         status;
```

Comment:
If you are using `MPI_STATUS_IGNORE` below instead, you don't need this variable.